connectors/zendesk.mdx

```diff
@@ -0,0 +1,29 @@
+---
+title: Zendesk Connector
+description: 'Access knowledge from Zendesk Articles'
+---
+
+## How it works
+The Zendesk Connector syncs all of the Articles in your company's subdomain.
+
+Some current limitations (a contributor is currently working on extending these):
```

Comment:
@yuhongsun96 Do you know if a contributor is still working on adding in the ability for the Zendesk connector to pick up tickets in addition to articles? If not, I'd like to pick up that work.

Reply:
Not that I know of, feel free to take it up!